src/main/resources/standardTestCases/exit.test

```diff
@@ -0,0 +1,13 @@
+name: exit
+testFormat: workflowsuccess
+backends: [Jes, Local]
```

Comment:
I thinks this is what you want but just to double check, setting this doesn't cause work to go to those backends, it just makes sure it will only allow it to go there